tests/parser/types/numbers/test_sqrt.py

```diff
@@ -0,0 +1,63 @@
+from decimal import (
+    ROUND_FLOOR,
+    Decimal,
+)
+
+
+def decimal_sqrt(val):
```

Comment:
I prefer to have a separate function for rounding
```suggestion
def decimal_round(val, places, rounding=ROUND_FLOOR):
  q = '0'
  if places != 0 : 
    q += '.' + '0' * places
  return val.quantize(Decimal(q), rounding=rounding)
```
That way it's easier to check at a glance the number of places being rounded

Reply:
I think your example is missing the .sqrt() ;)